src/yz_kv.erl

```diff
@@ -402,12 +402,25 @@ first_partition([{Partition, _}|_]) ->
 
 %% @private
 %%
-%% @doc Get the tree.
+%% @doc Get the tree if yz anti_entropy is enabled.
 %%
-%% NOTE: Relies on the fact that this code runs on the KV vnode
-%%       process.
 -spec get_tree(p()) -> tree() | not_registered.
 get_tree(Partition) ->
```

Comment:
I was just attempting to even avoid https://github.com/basho/yokozuna/blob/bugfix/zl/handle-yz-aae-disabled/src/yz_kv.erl#L424, but probably makes little difference (and in get_and_set, it'd be cleaner)